ariadne/executable_schema.py

```diff
@@ -341,9 +365,22 @@ def join_type_defs(type_defs: List[str]) -> str:
     return "\n\n".join(t.strip() for t in type_defs)
 
 
+def normalize_bindables(*bindables: SchemaBindables) -> List[SchemaBindable]:
+    normal_bindables: List[SchemaBindable] = []
+    for bindable in flatten_bindables(*bindables):
+        if isinstance(bindable, SchemaBindable):
+            normal_bindables.append(bindable)
+        elif issubclass(bindable, Enum):
+            normal_bindables.append(EnumType(bindable.__name__, bindable))
+        else:
+            raise ValueError(f"Unknown supported type: {repr(bindable)}")
```

Comment:
`"Unknown supported type"` looks strange to me, maybe `"Not supported type"` or `"Unknown type"`?